applications: nrf_desktop: Fix turning off HID keyboard LEDs (#5735)
nrfconnect/sdk-nrf: doc/nrf/releases/release-notes-changelog.rst

  * Updated documentation for :ref:`nrf_desktop_usb_state`.
  * Updated documentation with information about forwarding boot reports.
    See the documenation page of nRF Desktop's :ref:`nrf_desktop_hid_forward` for details.

>   * Fixed an issue causing HID keyboard LEDs to remain turned on after host disconnected if there is no other host connected.

Reviewer: ```suggestion
  * Fixed an issue that was causing the HID keyboard LEDs to remain turned on after host disconnection and no other hosts were connected.
```

Author: `while no other hosts were connected`?

Reviewer: Sounds good
